src/emailAlert.ts

```diff
@@ -0,0 +1,35 @@
+const sendgrid = require('@sendgrid/mail')
+
+import { error, log } from './debug'
+
+const api_key = process.env.SENDGRID_API_KEY
+const alert_email = process.env.ALERT_EMAIL
```

Comment:
This introduces yet another pattern of naming variables in this repository, I suggest using either `SCREAMING_SNAKE_CASE` or `pascalCase`, since both were already used in other places